default/methods/task/task/task_user_remove.py

```diff
@@ -3,23 +3,43 @@
 from shared.database.task.task_user import TaskUser
 
 
-@routes.route('/api/v1/project/<string:project_string_id>/task/user/remove/<int:task_user_id>', methods = ['DELETE'])
+@routes.route('/api/v1/project/<string:project_string_id>/task/<int:task_id>/user/remove', methods = ['POST'])
 @Project_permissions.user_has_project(Roles = ["admin", "Editor"], apis_user_list = ["api_enabled_builder"])
-def api_task_user_remove(project_string_id, task_user_id):
+def api_task_user_remove(project_string_id, task_id):
     with sessionMaker.session_scope() as session:
-        spec_list = []
+        spec_list = [
+            {'user_id': {
+            'required': False,
+            }},
+            {'relation': {
+                'required': True,
+                'kind': str,
+                'valid_values_list': ['reviewer', 'assignee']
+            }},
+        ]
+        print("remove")
+
+        result = {}
 
         log, input, untrusted_input = regular_input.master(request = request,
                                                            spec_list = spec_list)
         if len(log["error"].keys()) >= 1:
             return jsonify(log = log), 400
 
-        result, log = task_user_remove_core(
-            session = session,
-            task_user_id = task_user_id,
-            project_string_id = project_string_id,
-            log = log
-        )
+        for user in input['user_id']:
```

Comment:
Same as above comments. 

- Changing the `user_id` to `user_id_list`
- Moving the forloop logic into the core function to keep the API with just the above mentioned responsabilities.